tests/bluetooth/tester/src/btp/btp.h

```diff
@@ -75,8 +76,9 @@
 #define BTP_SERVICE_ID_TBS      0x1b
 #define BTP_SERVICE_ID_TMAP     0x1c
 #define BTP_SERVICE_ID_OTS      0x1d
+#define BTP_SERVICE_ID_PBP      0x1e
 
-#define BTP_SERVICE_ID_MAX	BTP_SERVICE_ID_OTS
+#define BTP_SERVICE_ID_MAX	BTP_SERVICE_ID_PBP
```

Comment:
What happened here?

Reply:
Increments the `BTP_SERVICE_ID_MAX`. Has to be done manually since it's not an enum that would increment this automatically :)